drivers/driver_linux.go

```diff
@@ -53,11 +53,11 @@ const (
 var (
 	// Slice of drivers that should be used in an order
 	priority = []string{
+		"overlay",
```

Comment:
There was no overlay2 in the list, and I am no expert on containers/storage.  My goal with this change is